docs/installation.md

```diff
@@ -2,20 +2,37 @@
 
 ## Installation
 
-First, make sure you have [created and activated](https://docs.python.org/3/library/venv.html) a Python3.8+ virtual environment.
+=== "UV"
 
-Then, run
-```console
-python -m pip install narwhals
-```
+    First, ensure you have installed [UV](https://github.com/astral-sh/uv), and make sure you have [created and activated](https://docs.astral.sh/uv/pip/environments/#python-environments) a Python 3.8+ virtual environment.
+
+    If you haven't, you can follow our guide [here](https://github.com/narwhals-dev/narwhals/blob/main/CONTRIBUTING.md#option-1-use-uv-recommended).
+
```

Comment:
The wording here is not very certain to me, perhaps it could be improved.

Reply:
I have a tiny suggestion to avoid using "here" (you can read more - [Don’t use "click here"](https://heyoka.medium.com/dont-use-click-here-f32f445d1021))

```suggestion
    First, ensure you have installed [UV](https://github.com/astral-sh/uv), and make sure you have [created and activated](https://docs.astral.sh/uv/pip/environments/#python-environments) a Python 3.8+ virtual environment.

    If you haven't, you can follow our [_setting up your environment_](https://github.com/narwhals-dev/narwhals/blob/main/CONTRIBUTING.md#option-1-use-uv-recommended) guide.

```

Reply:
Thank you for reminding me about `here`. I was completely unaware before!